lib/backpex/html/layout.ex

```diff
@@ -230,35 +230,60 @@ defmodule Backpex.HTML.Layout do
   Get the Backpex logo SVG.
   """
   @doc type: :component
-  attr(:class, :string,
-    required: false,
-    default: "",
-    doc: "class that will be added to the SVG element"
-  )
+  attr(:class, :string, required: false, default: nil, doc: "class that will be added to the SVG element")
 
   def backpex_logo(assigns) do
```

Comment:
I don't know if `layout.ex` is still the right place for the logo component. What's your opinion on that? @pehbehbeh

Reply:
I would remove the component completely and just use `priv/static/images/logo.svg` via `img` tags? Maybe move it to `demo/assets/images` (?) and adjust paths in README and docs.

We do not need to have a component with a dynamic class. We are not changing colors or anything inside the SVG. Size and spacing can be controlled from outside.

Reply:
> Maybe move it to demo/assets/images

We should not move it to the demo directory because then the image is not part of the Backpex package.

We use the logo in Backpex as well as in the demo. If we use `img` tags, we have to copy the SVG into both projects.

The logo is used here:
- [Demo: `index.html.heex#L5`](https://github.com/naymspace/backpex/blob/0.4.0/demo/lib/demo_web/live/home_live/index.html.heex#L5)
- [Demo: `index.html.heex#L114`](https://github.com/naymspace/backpex/blob/0.4.0/demo/lib/demo_web/live/home_live/index.html.heex#L114)
- [Backpex: `layout.ex#L218`](https://github.com/naymspace/backpex/blob/0.4.0/lib/backpex/html/layout.ex#L218)